packages/cozy-harvest-lib/src/locales/en.json

```diff
@@ -489,7 +489,16 @@
   "datacards": {
     "trips": {
       "title": "Last trips",
-      "caption": "See your 5 most recent trips retrieved on %{konnectorName}"
+      "caption": "See your 5 most recent trips retrieved on %{konnectorName}",
+      "modes": {
+        "AIR_OR_HSR": "Plane",
+        "BICYCLING": "Bike",
+        "CAR": "Car",
+        "SUBWAY": "Subway",
+        "TRAIN": "Train",
+        "UNKNOWN": "",
+        "WALKING": "Walking"
```

Comment:
How did you find the available modes?

Reply:
By looking at the data

Reply:
Looks like the data was complete: https://github.com/e-mission/e-mission-server/blob/0475b456db197edacb524c5ab7a1124cd33b5333/emission/core/wrapper/modeprediction.py#L16